systemtests/scripts/run_test_kubernetes.sh

```diff
@@ -32,10 +34,16 @@ mkdir -p api-server-cert/
 openssl req -new -x509 -batch -nodes -days 11000 -subj "/O=io.enmasse/CN=api-server.${KUBERNETES_NAMESPACE}.svc.cluster.local" -out api-server-cert/tls.crt -keyout api-server-cert/tls.key
 kubectl create secret tls api-server-cert --cert=api-server-cert/tls.crt --key=api-server-cert/tls.key
 
+if [ "$ONLY_PULL_ONCE" = "true" ]; then
+	# disable loading images always when running on travis
+	# we don't get refresh images, so we can speed this up
+	sed -i "s/imagePullPolicy: Always/imagePullPolicy: IfNotPresent/g" ${ENMASSE_DIR}/install/*/*/*.yaml
```

Comment:
Perhaps instead set  IMAGE_PULL_POLICY=IfNotPresent .travis.yml

Reply:
Will that override the makefile?

Reply:
Ok, I changes this to "common.sh" when it is a PR